$NetBSD: patch-lib_libpe_Makefile,v 1.2 2014/12/09 14:37:06 khorben Exp $

Fixed generation and installation of the libpe shared library on NetBSD

--- lib/libpe/Makefile.orig	2013-12-27 10:39:52.000000000 +0000
+++ lib/libpe/Makefile
@@ -70,6 +70,8 @@ libpe: CPPFLAGS += -D_GNU_SOURCE
 libpe: $(libpe_OBJS)
 ifeq ($(PLATFORM_OS), Linux)
 	$(LINK) -shared -Wl,-soname,$(LIBNAME).so.1 $(LDFLAGS) -o $(LIBNAME).so $^
+else ifeq ($(PLATFORM_OS), NetBSD)
+	$(LINK) -shared -Wl,-soname,$(LIBNAME).so.1 $(LDFLAGS) -o $(LIBNAME).so $^
 else ifeq ($(PLATFORM_OS), Darwin)
 	$(LINK) -headerpad_max_install_names -dynamiclib \
 		-flat_namespace -install_name $(LIBNAME).$(VERSION).dylib \
@@ -88,6 +90,10 @@ ifeq ($(PLATFORM_OS), Linux)
 	$(INSTALL_DATA) $(LIBNAME).so $(DEST)/$(LIBNAME).so.$(VERSION)
 	cd $(DEST); $(SYMLINK) $(LIBNAME).so.$(VERSION) $(LIBNAME).so
 	cd $(DEST); $(SYMLINK) $(LIBNAME).so.$(VERSION) $(LIBNAME).so.1
+else ifeq ($(PLATFORM_OS), NetBSD)
+	$(INSTALL_DATA) $(LIBNAME).so $(DEST)/$(LIBNAME).so.$(VERSION)
+	cd $(DEST); $(SYMLINK) $(LIBNAME).so.$(VERSION) $(LIBNAME).so
+	cd $(DEST); $(SYMLINK) $(LIBNAME).so.$(VERSION) $(LIBNAME).so.1
 else ifeq ($(PLATFORM_OS), Darwin)
 	$(INSTALL_DATA) $(LIBNAME).dylib $(DEST)/$(LIBNAME).$(VERSION).dylib
 	cd $(DEST); $(SYMLINK) $(LIBNAME).$(VERSION).dylib $(LIBNAME).dylib
@@ -102,6 +108,8 @@ installdirs:
 strip-binaries:
 ifeq ($(PLATFORM_OS), Linux)
 	$(STRIP) $(LIBNAME).so
+else ifeq ($(PLATFORM_OS), NetBSD)
+	$(STRIP) $(LIBNAME).so
 else ifeq ($(PLATFORM_OS), Darwin)
 	$(STRIP) $(LIBNAME).dylib
 else ifeq ($(PLATFORM_OS), CYGWIN)
